pkgs/development/tools/faq/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, jq
+, oniguruma
+}:
+
+buildGoModule rec {
+  pname = "faq";
+  version = "0.0.7";
```

> Unfortunately 0.0.7 does not build out of the box - vendor issues. There is no release after that, but only few commits on top.

Then if so, use:

```suggestion
  version = "unstable-2022-01-09";
```

Where the date is the date of the commit you are fetching. Don't forget to change the commit message, and update the PR title.

---

And also, it'd be nice to mention in a comment near the `version = ` line why don't you take the last release version.